src/openrct2/world/sprite.c

```diff
@@ -951,6 +951,50 @@ sint32 check_for_sprite_list_cycles(bool fix)
     return -1;
 }
 
+/**
+ * Finds and fixes null sprites that are not reachable via SPRITE_LIST_NULL list.
+ *
+ * @return count of disjoint sprites found
+ */
+sint32 fix_disjoint_sprites()
+{
+    // Find reachable sprites
+    bool reachable[MAX_SPRITES] = {};
```

Comment:
Good question, but it refuses to build with MSVC (see Appveyor) without it. It's stupid, but then MSVC often is.